### PR TITLE
Add java snippets

### PR DESCRIPTION
**Pull request by outoftardis**

### CODE REVIEW

**Review by 6r1d — APPROVED** [review]
LGTM so far, although why is it a draft?

File: snippet_sources.json [hunk 1]
```diff
@@ -109,5 +109,17 @@
   {
     "src": "https://raw.githubusercontent.com/hyperledger/iroha-javascript/iroha2/packages/docs-recipes/src/9.blocks-stream.ts",
     "filename": "js-sdk-9-blocks-stream.ts"
+  },
+  {
+    "src": "https://raw.githubusercontent.com/hyperledger/iroha-java/iroha2-dev/modules/test-tools/src/main/kotlin/jp/co/soramitsu/iroha2/testengine/IrohaConfig.kt",
+    "filename": "IrohaConfig.kt"
+  },
+  {
+    "src": "https://raw.githubusercontent.com/hyperledger/iroha-java/doc-snippets/modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt",
```
Comment (outoftardis) [inline snippet_sources.json:118]:
```suggestion
    "src": "https://raw.githubusercontent.com/hyperledger/iroha-java/iroha2-dev/modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt",
```

**Review by 6r1d — APPROVED** [review]
LGTM

File: src/guide/kotlin-java.md [hunk 2]
```diff
@@ -367,200 +219,4 @@ submitted was processed correctly and provide feedback to the end-user.
...
-
-        final Value value = future.get(10, TimeUnit.SECONDS);
-        Assertions.assertEquals(
-            ((Value.String) value).getString(),
-            assetValue.getString()
-        );
-    }
-}
-```
+<<<@/snippets/JavaTest.java{java}
```
Comment (0x009922) [inline src/guide/kotlin-java.md:222]:
```suggestion
<<<@/snippets/JavaTest.java
```

Here it is unnecessary, I think.

File: snippet_sources.json [hunk 3]
```diff
@@ -109,5 +109,17 @@
   {
     "src": "https://raw.githubusercontent.com/hyperledger/iroha-javascript/iroha2/packages/docs-recipes/src/9.blocks-stream.ts",
     "filename": "js-sdk-9-blocks-stream.ts"
+  },
+  {
+    "src": "https://raw.githubusercontent.com/hyperledger/iroha-java/iroha2-dev/modules/test-tools/src/main/kotlin/jp/co/soramitsu/iroha2/testengine/IrohaConfig.kt",
+    "filename": "IrohaConfig.kt"
```
Comment (0x009922) [inline snippet_sources.json:115]: You can either omit `filename` at all (as it is the same as in `src`), or specify it as `IrohaConfig.kotlin`, thus you will not need to write `{kotlin}` in snippet import syntax.

**Review by 0x009922 — APPROVED** [review]

File: snippet_sources.json (hunk 3 above)
Comment (outoftardis) [inline snippet_sources.json:115]:
```suggestion
    "filename": "IrohaConfig.kotlin"
```

File: src/guide/kotlin-java.md [hunk 4]
```diff
@@ -75,27 +75,7 @@ resident `config.json` in the working directory.
...
-
-class IrohaConfig(
-    var networkToJoin: Network = newNetwork(),
-    var logConsumer: Consumer<OutputFrame> = Slf4jLogConsumer(getLogger(IrohaContainer::class.java)),
-    var genesis: Genesis = Genesis(RawGenesisBlock(listOf(GenesisTransaction(listOf())))),
-    var shouldCloseNetwork: Boolean = true,
-    var imageTag: String = IrohaContainer.DEFAULT_IMAGE_TAG
-)
-```
+<<<@/snippets/IrohaConfig.kt{kotlin}
```
Comment (outoftardis) [inline src/guide/kotlin-java.md:78]:
```suggestion
<<<@/snippets/IrohaConfig.kotlin
```

**Pull request merged**